composer.json

```diff
@@ -42,7 +43,6 @@
     "bin": [
         "bin/changelog-generator"
     ],
-    "platform": {
-        "php": "7.1.x"
```

Comment:
This thing has different purpose than `require.php` - it tells composer to install dependencies compatible with PHP 7.1, whatever PHP versions are allowed. It's not a replacement of `require.php`.

Reply:
This is what this is for, right? https://github.com/jwage/changelog-generator/blob/master/build-phar.sh#L6

Reply:
Yes, one of those use cases. :)